pom.xml

```diff
@@ -17,15 +17,14 @@
   <url>https://github.com/jenkinsci/p4-plugin</url>
 
   <properties>
-    <jenkins.version>2.346.3</jenkins.version>
-    <java.level>8</java.level>
+    <jenkins.version>2.375.3</jenkins.version>
   </properties>
 
   <scm>
-    <connection>scm:git:git://github.com/jenkinsci/${project.artifactId}-plugin.git</connection>
+    <connection>scm:git:https://github.com/jenkinsci/${project.artifactId}-plugin.git</connection>
     <developerConnection>scm:git:git@github.com:jenkinsci/${project.artifactId}-plugin.git</developerConnection>
     <url>https://github.com/jenkinsci/${project.artifactId}-plugin</url>
-    <tag>p4-1.14.2</tag>
+    <tag>HEAD</tag>
```

Comment:
We would like to keep the tag as it is e.g. <tag>p4-1.14.2</tag>, this is required for the release process.